test/java/clojure/lang/UtilTest.java

```diff
@@ -0,0 +1,227 @@
+
+    Assert.assertTrue(Util.equivPred(k1).equiv(k1, k2));
+
+    k2 = new ArraySeq(new String[2], 0);
+
+    Assert.assertFalse(Util.equivPred(k1).equiv(k1, k2));
+  }
+
+
+  @Test
+  public void testEquivPredNull() {
+    Object k1 = null;
+    Object k2 = null;
+
+    Assert.assertTrue(Util.equivPred(k1).equiv(k1, k2));
+
+    k2 = "foo";
+
+    Assert.assertFalse(Util.equivPred(k1).equiv(k1, k2));
+  }
+
+  @Test
+  public void testEquivPredEquals() {
+    Object k1 = "foo";
+    Object k2 = "foo";
+
+    Assert.assertTrue(Util.equivPred(k1).equiv(k1, k2));
+
+    k2 = "Bar";
+
+    Assert.assertFalse(Util.equivPred(k1).equiv(k1, k2));
+  }
+
+  @Test
+  public void testEquivPredNumber() {
+    Object k1 = 1;
+    Object k2 = 1;
+
+    Assert.assertTrue(Util.equivPred(k1).equiv(k1, k2));
+
+    k1 = 0;
+
+    Assert.assertFalse(Util.equivPred(k1).equiv(k1, k2));
+
+    k2 = null;
+
+    Assert.assertFalse(Util.equivPred(k1).equiv(k1, k2));
+
+    k1 = null;
+    k2 = 0;
+
+    Assert.assertFalse(Util.equivPred(k1).equiv(k1, k2));
+  }
+
+  @Test
+  public void testHashCombine() {
+    Assert.assertEquals(0, Util.hashCombine(908_697_334, 0));
+    Assert.assertEquals(483896201, Util.hasheq(BigInt.fromLong(32L)));
+    Assert.assertEquals(0, Util.hasheq(null));
+    Assert.assertEquals(875635954, Util.hasheq(64));
+  }
+
+  @Test
+  public void testHash() {
+    Assert.assertEquals(0, Util.hash(null));
+    Assert.assertEquals(64, Util.hash(64));
+    Assert.assertEquals(-1268909715, Util.hash("fooBar"));
+  }
+
+  @Test
+  public void testIdentical() {
+    Assert.assertTrue(Util.identical(0, 0));
+    Assert.assertFalse(Util.identical(0, 1));
+    Assert.assertFalse(Util.identical(new Integer(0), new Integer(0)));
+  }
+
+  @Test
+  public void testIsInteger() {
+    Assert.assertTrue(Util.isInteger(0));
+    Assert.assertTrue(Util.isInteger(32L));
+    Assert.assertTrue(Util.isInteger(BigInt.fromLong(32L)));
+    Assert.assertTrue(Util.isInteger(new BigInteger("1".getBytes())));
+    Assert.assertFalse(Util.isInteger("fooBar"));
+  }
+
+  @Test
+  public void testIsPrimitive() {
+    Assert.assertTrue(Util.isPrimitive(int.class));
+    Assert.assertFalse(Util.isPrimitive(Integer.class));
+  }
+
+  @Test
+  public void testPcequiv() {
+    final PersistentVector k1 = new PersistentVector(0, 0, null, null);
+    final PersistentVector k2 = new PersistentVector(0, 0, null, null);
+
+    Assert.assertFalse(Util.pcequiv(k1, null));
+    Assert.assertFalse(Util.pcequiv(0, k2));
+    Assert.assertTrue(Util.pcequiv(k1, k2));
+  }
+
+  @Test
+  public void testRet1() {
+    Assert.assertEquals(0, Util.ret1(0, 0));
+  }
+
+  @Test
+  public void testRuntimeException() {
+    final RuntimeException runtimeException = Util.runtimeException(
+            "NullPointerException",
+            new Throwable()
+    );
+
+    Assert.assertEquals(
+            "NullPointerException",
+            runtimeException.getMessage()
```

Comment:
Same here

Reply:
if it fits within the 80 character limit then it can be done on two lines and still look clear to the user.